openbb_terminal/futures/futures_controller.py

```diff
@@ -74,7 +74,15 @@ def __init__(self, queue: List[str] = None):
             self.choices["support"] = self.SUPPORT_CHOICES
             self.choices["about"] = self.ABOUT_CHOICES
 
-            self.choices["historical"] = {c: None for c in self.all_tickers}
+            self.choices["historical"] = {
```

Comment:
This will not have autocomplete for users that just do `historical TICKER`, so we should be doing something like forex does:
```
choices["load"] = {c: {} for c in FX_TICKERS}
choices["load"]["--ticker"] = {c: {} for c in FX_TICKERS}
choices["load"]["-t"] = "--ticker"
choices["load"]["--resolution"] = {c: {} for c in self.RESOLUTION}
choices["load"]["-r"] = "--resolution"
choices["load"]["--interval"] = {
    c: {} for c in SOURCES_INTERVALS["YahooFinance"]
}
choices["load"]["--start"] = None
choices["load"]["-s"] = "--start"
```

Reply:
oh oops, i was going to do that and forgot to add it! will do